src/app/community-page/sub-collection-list/community-page-sub-collection-list.component.spec.ts

```diff
@@ -122,6 +130,25 @@ describe('CommunityPageSubCollectionList Component', () => {
 
   themeService = getMockThemeService();
 
+  const linkHeadService = jasmine.createSpyObj('linkHeadService', {
+    addTag: ''
+  });
+
+  const groupDataService = jasmine.createSpyObj('groupsDataService', {
+    findAllByHref: createSuccessfulRemoteDataObject$(createPaginatedList([])),
+    getGroupRegistryRouterLink: '',
+    getUUIDFromString: '',
```

Comment:
The reason that all these files had spys added is because of where the rss-feed component is located since it is located in the pagination service class which is referenced by a bunch of other components those components then need to be updated with spys when they are mocked or angular will complain that im not passing the spys down correctly.